recipes/boost/all/conanfile.py

```diff
@@ -1345,7 +1345,7 @@ def _toolset_tag(self):
         if self._is_msvc:
             toolset_version = self._toolset_version.replace(".", "")
         else:
-            toolset_version = str(Version(self.settings.compiler.version).major)
+            toolset_version = str(Version(self.settings.compiler.version).major(fill=False))
```

Comment:
@uilianries please note that this is going to break in next release because of https://github.com/conan-io/conan/pull/11823

Reply:
@lasote , what works from `1.50` (current release in CCI) to `1.52` and `2.x`? This is something we can add to the linter asap so we prevent changing recipes now and afterward again. This is the kind of change/fix that will probably spread to other recipes.

Thanks!

Reply:
Could we implement dummy `fill` args for `conan.tools.scm.Version` properties?

Otherwise seems like we'd have to set `required_conan_version = ">=1.52.0"` here which would be pretty bad.

Reply:
vale, I gonna open a new PR to avoid it.

Reply:
@uilianries , please, open a PR reverting (?) the changes, but please mention and wait for @lasote or @jcar87 to say something about a possible _working_ syntax. Thanks!

Reply:
@jgsogo doing in 3..2..1

Reply:
Done, new PR available: https://github.com/conan-io/conan-center-index/pull/12131